cram_common/cram_object_knowledge/src/household.lisp

```diff
@@ -325,3 +325,440 @@
+        )
+    (desig:a location
+             (pose ?pose)
+             ;; (left-of (desig:an object (type ?other-object)))
+             ;; (far-from (desig:an object (type ?other-object)))
+             ;; (for (desig:an object (type ?object-type)))
+             ;; (on (desig:an object
+             ;;               (type counter-top)
+             ;;               (urdf-name kitchen-island-surface)
+             ;;               (owl-name "kitchen_island_counter_top")
+             ;;               (part-of ?kitchen-name)))
+             ;; (side back)
+             )))
+
+(defmethod man-int:get-object-likely-destination :heuristics 20
+    ((?kitchen-name (eql :kitchen))
+     ?human-name 
+     (?context (eql :table-cleaning))
+     (?object-type (eql :breakfast-cereal)))
+
+  (desig:a location
+           (on (desig:an object
+                         (type area-sink)
+                         (urdf-name sink-area-sink)
+                         (owl-name "kitchen_sink_area_sink")
+                         (part-of ?kitchen-name)))
+           (for (desig:an object (type ?object-type)))))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;; milk ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(defmethod man-int:get-object-likely-destination :heuristics 20 
+    ((?kitchen-name (eql :kitchen))
+     ?human-name 
+     (?context (eql :table-setting))
+     (?object-type (eql :milk)))
+
+  (let ((?other-object :bowl))
+    (desig:a location
+             (left-of (desig:an object (type ?other-object)))
+             (far-from (desig:an object (type ?other-object)))
+             (for (desig:an object (type ?object-type))))))
+
+(defmethod man-int:get-object-likely-destination :heuristics 20
+    ((?kitchen-name (eql :kitchen))
+     ?human-name 
+     (?context (eql :table-cleaning))
+     (?object-type (eql :milk)))
+
+  (desig:a location
+           (on (desig:an object
+                         (type area-sink)
+                         (urdf-name sink-area-sink)
+                         (owl-name "kitchen_sink_area_sink")
+                         (part-of ?kitchen-name)))
+           (for (desig:an object (type ?object-type)))))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;; bottle ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(defmethod man-int:get-object-likely-destination :heuristics 20 
+    ((?kitchen-name (eql :kitchen))
+     ?human-name 
+     (?context (eql :table-setting))
+     (?object-type (eql :bottle)))
+
+  (desig:a location
+           (on (desig:an object
+                         (type counter-top)
+                         (urdf-name kitchen-island-surface)
+                         (owl-name "kitchen_island_counter_top")
+                         (part-of ?kitchen-name)))
+           (for (desig:an object (type ?object-type)))))
+
+(defmethod man-int:get-object-likely-destination :heuristics 20
+    ((?kitchen-name (eql :kitchen))
+     ?human-name 
+     (?context (eql :table-cleaning))
+     (?object-type (eql :bottle)))
+
+  (desig:a location
```

Comment:
Throwing bottles into the sink is kind of weird. So I'd say either trash or back onto the sink area surface.